bedrock/firefox/firefox_details.py

```diff
@@ -341,7 +353,7 @@ class FirefoxAndroid(_ProductDetails):
         'x86': archive_url_base + '-%s/fennec-%s.multi.android-i386.apk',
     }
 
-    def platforms(self, channel='release'):
+    def platforms(self, channel='release', classified=False):
```

Comment:
Could we add a comment here explaining why this method has an unused `classified` parameter?